HISTORY.md

```diff
@@ -33,7 +33,7 @@ This release only changed documentation.
 2.7.1 / 2015-05-28
 ==================
 
-  * Move module into expressjs organization
+  * Move module into Express.js organization
```

Comment:
This is because the github organization is literally called "expressjs". Can you revert this edit?